1xx-enable-ssl-for-kafka-bridge.md

```diff
@@ -0,0 +1,166 @@
+# Support TLS/SSL on the HTTP interface 
+
+This proposal adds an option to enable TLS encryption on HTTP interface so that requests from clients to the Kafka Bridge can be sent with encryption.
+
+## Motivation
+
+The connection between HTTP Bridge and Kafka cluster can be secured however, currently there is no support for securing the connections to HTTP Bridge from clients. The current recommendation is to use Network policies and firewalls, reverse proxies such as OAuth 2.0 or API gateways to secure the client connections. This requires users to make additional configurations but does not provide the best user experience.  
+
+## Proposal
+
+This proposal adds new configurations to enable SSL server and specify locations of certificate and key files in PEM format so that client connectionts to HTTP Bridge is encryped. The new configurations will be added to the current `http` prefixed configurations:
+- http.ssl.enable
+- http.ssl.keystore.location
+- http.ssl.keystore.key.location
+- http.ssl.protocol
+- http.ssl.enabled.protocols
+
+When `http.ssl.enable` is set to true, Kafka Bridge server would be started with ssl enabled and the key and certificate loaded from the locations defined by `http.ssl.keystore.location` and `http.ssl.keystore.key.location` configurations. If `http.port` configuration is not set, the server port will be set to `8443` by default. This will allow users to make HTTPS connections from their clients, and the server would not accept unsecured connections.
+
+If `http.ssl.protocol` is configured, the protocol will be added to the current or default list of enabled protocols. If `http.ssl.enabled.protocols` is configured, the list will replace the default list of enabled secure transport protocols that Vertx sets.
```

Comment:
Aren't there some existing Vert.x options to configure this? If not, do we need the same options also for cipher suites? As that is another common thing users configure.